slackdump/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .command import main
```

Comment:
Is there any particular reason for putting `.` in the import statements? When we create a package everything is bundled together, I dont think it is required.

Reply:
It's required in python3